packages/syft/src/syft/telemetry.py

```diff
@@ -1,6 +1,19 @@
 # stdlib
 import os
+import sys
+from typing import Callable
 from typing import Optional
+from typing import TypeVar
+from typing import Union
+
+if sys.version_info >= (3, 10):
+    # stdlib
```

Comment:
I imagine this is the same as the other PR, since we want typing extensions for older python we should just use it until we deprecate a particular version such as 3.8.

Reply:
Yup I also replied under the other comment. This is more like self documentation to remind us to move from `typing_extensions` to `typing` when the minimum required version Python supports the feature. It's just a convention I saw widely used for using standard library backports.